tests/unit/fraud-prevention/test-class-fraud-prevention-service.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Class Fraud_Prevention_Service_Test
+ *
+ * @package WooCommerce\Payments\Tests
+ */
+
+use WCPay\Fraud_Prevention\Fraud_Prevention_Service;
+
+/**
+ * Fraud_Prevention_Service_Test unit tests.
+ */
+class Fraud_Prevention_Service_Test extends WP_UnitTestCase {
+
+	/**
+	 * WC_Session mock object.
+	 *
+	 * @var PHPUnit_Framework_MockObject_MockObject|WC_Session
+	 */
+	private $session_mock;
+
+	/**
+	 * The service under test.
+	 *
+	 * @var Fraud_Prevention_Service
+	 */
+	private $fraud_prevention_service;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->session_mock = $this->getMockBuilder( 'WC_Session' )
+			->getMock();
+
+		Fraud_Prevention_Service::set_instance( null );
+		$this->fraud_prevention_service = Fraud_Prevention_Service::get_instance( $this->session_mock );
+	}
+
+	public function test_it_checks_if_enabled() {
+		// TODO: depends on #3994.
+		$this->markTestSkipped( 'depends on #3994' );
```

Comment:
I think, now we can implement the test here, #3994 is in develop branch

Reply:
+1